app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,7 @@
 class PostsController < ApplicationController
+  # load_and_authorize_resource
```

Comment:
- [ ] Why did you comment load_and_authorize? To run your Ability, you need to have this.